main_test.go

```diff
@@ -213,6 +214,33 @@ func TestHealthIntegration(t *testing.T) {
 	main()
 }
 
+func TestBenchmarkIntegration(t *testing.T) {
+	origArgs := os.Args
+	defer func() { os.Args = origArgs }()
+
+	const numServers = 5
```

Comment:
Why make this a constant here? It's not used anywhere else. Just for readability?